openverse_catalog/dags/providers/provider_api_scripts/provider_data_ingester.py

```diff
@@ -247,6 +272,20 @@ def ingest_records(self, **kwargs) -> None:
         if error_summary := self._get_ingestion_errors():
             raise error_summary
 
+    def _should_skip_ingestion_error(self, error: Exception) -> bool:
+        """Determine whether an error should be skipped."""
+        if self.skip_all_ingestion_errors:
+            return True
+
+        # Look for predicates that match either the error message or class
+        error_str = f"{error.__class__.__name__} {error}"
```

Comment:
I wonder, do you think we could just use the exception's representation here instead?
```python
[ins] In [1]: a = ValueError("Something went horribly wrong")

[ins] In [3]: repr(a)
Out[3]: "ValueError('Something went horribly wrong')"

[ins] In [4]: print(a)
Something went horribly wrong

[ins] In [5]: import airflow.exceptions

[ins] In [6]: airflow.exceptions.AirflowException("Foobar!")
Out[6]: AirflowException('Foobar!')

[ins] In [7]: b = _

[ins] In [8]: repr(b)
Out[8]: "AirflowException('Foobar!')"
```

Reply:
Definitely!